pkg/v1/remote/transport/scheme.go

```diff
@@ -15,28 +15,10 @@
 package transport
 
 import (
-	"regexp"
-	"strings"
-
 	"github.com/google/go-containerregistry/pkg/name"
 )
 
-// Detect more complex forms of local references.
-var reLocal = regexp.MustCompile(`.*\.local(?:host)?(?::\d{1,5})?$`)
-
-// Detect the loopback IP (127.0.0.1)
-var reLoopback = regexp.MustCompile(regexp.QuoteMeta("127.0.0.1"))
-
 // Scheme returns https scheme for all the endpoints except localhost.
 func Scheme(reg name.Registry) string {
```

Comment:
This func doesn't seem to add much anymore, can we just update callers to call `reg.Scheme` directly? (Could also be a separate PR)

Reply:
done, but I'm afraid to break for external uses